examples/tracing/ddos_detector.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
+#
+# ddos_detector.py	DDOS dectection system.
+#		
+#
+# Written as a basic networking example of using ePBF to detect a potential DDOS attack against a system.
+#
+#
+# Copyright (c) 2019 Jugurtha BELKALEM.
+# Licensed under the Apache License, Version 2.0 (the "License")
+#
+# 14-Jan-2019 Jugurtha BELKALEM Created this.
+
+from bcc import BPF
+import ctypes as ct
+prog = """
+#include <linux/skbuff.h>
+#include <uapi/linux/ip.h>
+
+/* If We receive more than 100 succesive packets with a difference of timestamp between each one of them is less than 1000000ns - Trigger ALERT */
+#define MAX_NB_PACKETS 100
+#define LEGAL_DIFF_TIMESTAMP_PACKETS 1000000
+
+BPF_HASH(received_packets);
+
+// define C structure
+struct detectionTimestamp {
+    u64 ts;
+};
+
+// use perf buffer (avoid using /sys/kernel/debug/tracing/trace_pipe)
+BPF_PERF_OUTPUT(events);
+
+int detect_ddos(struct pt_regs *ctx, struct sk_buff *skb, struct net_device *dev, struct packet_type *pt, struct net_device *orig_dev){
```

Comment:
For unused parameters like "pt", you can just do "void *pt" to avoid compilation warning.